src/clj/rems/api/applications.clj

```diff
@@ -230,6 +229,7 @@
     (command-endpoint :application.command/change-resources commands/ChangeResourcesCommand)
     (command-endpoint :application.command/close commands/CloseCommand)
     (command-endpoint :application.command/decide commands/DecideCommand)
+    (command-endpoint :application.command/delete commands/DeleteCommand)
```

Comment:
It would be nice to have in the swagger something about these, like that this is only for drafts (that was in the earlier directy endpoint) and only for the applicant. We don't really have that option now.

Reply:
Good point